doc/api/url.md

```diff
@@ -1251,13 +1260,22 @@ console.log(url.format(myURL, { fragment: false, unicode: true, auth: false }));
 // Prints 'https://測試/?abc'
 ```
 
-### `url.pathToFileURL(path)`
+### `url.pathToFileURL(path[, options])`
 
 <!-- YAML
 added: v10.12.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/52509
+    description: The `options` argument can now be used to
+                 determine how to parse the `path` argument.
 -->
 
 * `path` {string} The path to convert to a File URL.
+* `options` {Object}
+  * `windows` {boolean|undefined} `true` if the `path` should be
+    treated as a windows filepath, `false` otherwise.
+    **Default:** `undefined`.
```

Comment:
The implementation suggests that by default (if `windows` is `undefined` or `null`), the function will fallback to old behaviour (autodetection, depending on platform). It's probably worth mentioning here.

Reply:
*(resolved locally)*